src/app/meetups/add-meetups/meetups-add.component.ts

```diff
@@ -120,4 +136,25 @@ export class MeetupsAddComponent implements OnInit {
     this.router.navigate([ '/meetups' ]);
   }
 
+  isClassDay(day) {
+    return this.meetupFrequency.includes(day) ? true : false;
+  }
+
+  onDayChange(day: string, isChecked: boolean) {
+    if (isChecked) {
+      // add to day array if checked
+      if (!this.meetupFrequency.includes(day)) {
+        this.meetupFrequency.push(day);
+      }
+    } else {
+      // remove from day array if unchecked
+      const index = this.meetupFrequency.findIndex(x => x.value === day);
+      this.meetupFrequency.splice(index);
+    }
+  }
+
+  toggleDaily(val) {
+    val ? this.showDaysCheckBox = val : this.showDaysCheckBox = val;

```

Comment:
@Rupesh87 A couple of things I want to point out here:

1. Please do not use the ternary operator to run different code blocks.  It's easier to read our code if we only use it to assign a variable different values based on a condition.
2. In addition, you also have the same exact code on either side of the ternary operator.  There is no need for a conditional statement here at all.

Reply:
thanks @paulbert